docs/config/README.md

```diff
@@ -1760,6 +1761,42 @@ The module will be shown if any of the following conditions are met:
 format = "via [🐪 $version]($style) "
 ```
 
+## OpenStack
+
+The `openstack` module shows the current OpenStack cloud and project . This is based on
+`OS_CLOUD` env var with `clouds.yaml` file from any of the [default locations](https://docs.openstack.org/python-openstackclient/latest/configuration/index.html#configuration-files).
```

Comment:
From looking at the code it seems this module is only active if the `OS_CLOUD` env var is set, is this correct? If so might be worth add a note here to explain that.

Reply:
Fixed on my side. Will push later.